### ISSUE TITLE
let's talk

### ISSUE THREAD

**Issue by amir-arad** [tdelc/EmptyEpsilon#105]
I'd love to PM about sci-fi larps using EE when you're into it. greenshade on gee mail.

**Comment by tdelc** [comment]
Yes of course. I'm currently busy for our sci larps, Point Zero (http://quantum-games.be/), so I have a lot of things to do with EE.
If we can help and discuss each other, i could be great :)

**Comment by amir-arad** [comment]
wow, that's really cool!

**Comment by tdelc** [comment]
Hi Amir, my larp is done so I have now more time to discuss. I am currently trying to restart from the vanilla game to apply my previous features and yours, I will PR your sector system (simplify my myself ^^) to the main git this week.

(a video example of my modifications of your work : [Capture d'écran vidéo de 25-05-2020 10:05:51.webm.zip](https://github.com/tdelc/EmptyEpsilon/files/4676519/Capture.d.ecran.video.de.25-05-2020.10.05.51.webm.zip))

I also tested your WARP terrain system, it is awesome !!! I will certainly keep some of this feature to a future project.

We discuss with the french community to create a LARP version of Empty Epsilon, closest as possible to the main build (and contributing to it) but with more screen for LARP game. I think we could help each other for this.

So hope to discuss together, I am not sure I can discuss by vocal, because of my level of english, but why not, here or in the discord. Bye !
